### ISSUE TITLE
[Wrath Beta] Multiple localization errors in zhTW client

### ISSUE THREAD

**Issue by siweia** [Stanzilla/WoWUIBugs#247]
**1. CR_DODGE_TOOLTIP, CR_PARRY_TOOLTIP, CR_BLOCK_TOOLTIP, RESILIENCE_TOOLTIP, DEFAULT_STATDEFENSE_TOOLTIP
All of them using '%。2f%%' instead of '%.2f%%', cause error while opening PaperDollFrame.**
```
Interface_Wrath\FrameXML\PaperDollFrame.lua:706: invalid option in `format'
Count: 1

Call Stack:
[string "=[C]"]: ?
[string "=[C]"]: in function `format'
[string "@Interface_Wrath\FrameXML\PaperDollFrame.lua"]:706: in function `PaperDollFrame_SetDefense'
[string "@Interface_Wrath\FrameXML\PaperDollFrame.lua"]:459: in function `UpdatePaperdollStats'
[string "@Interface_Wrath\FrameXML\PaperDollFrame.lua"]:399: in function `PaperDollFrame_UpdateStats'
[string "@Interface_Wrath\FrameXML\PaperDollFrame.lua"]:218: in function <Interface_Wrath\FrameXML\PaperDollFrame.lua:175>

```

Temporary solution:
```
CR_DODGE_TOOLTIP = gsub(CR_DODGE_TOOLTIP, "。2", ".2")
CR_PARRY_TOOLTIP = gsub(CR_PARRY_TOOLTIP, "。2", ".2")
CR_BLOCK_TOOLTIP = gsub(CR_BLOCK_TOOLTIP, "。2", ".2")
RESILIENCE_TOOLTIP = gsub(RESILIENCE_TOOLTIP, "。2", ".2")
DEFAULT_STATDEFENSE_TOOLTIP = gsub(DEFAULT_STATDEFENSE_TOOLTIP, "。2", ".2")
```

**2. Error when TalentUI loaded.**
```
AddOns\Blizzard_TalentUI\Localization.lua:10: attempt to index global 'PlayerTalentFrameTalentPoints' (a nil value)
Count: 1

Call Stack:
[string "=[C]"]: ?
[string "@AddOns\Blizzard_TalentUI\Localization.lua"]:10: in main chunk
[string "=[C]"]: in function `LoadAddOn'
[string "@Interface_Wrath\FrameXML\UIParent.lua"]:382: in function `UIParentLoadAddOn'
[string "@Interface_Wrath\FrameXML\UIParent.lua"]:449: in function `TalentFrame_LoadUI'
[string "@Interface_Wrath\FrameXML\UIParent.lua"]:566: in function `ToggleTalentFrame'
[string "TOGGLETALENTS"]:1: in function <[string "TOGGLETALENTS"]:1>
```

Temporary solution:
```
if not PlayerTalentFrameTalentPoints then
	PlayerTalentFrameTalentPoints = CreateFrame("Frame")
end
if not PlayerTalentFrameSpentPoints then
	PlayerTalentFrameSpentPoints = CreateFrame("Frame")
end
```

**3. Error when InspectUI loaded. This seems to be unfixed for years. https://github.com/Stanzilla/WoWUIBugs/issues/190**
```
AddOns\Blizzard_InspectUI\Localization.lua:8: attempt to index global 'InspectTalentFrameSpentPoints' (a nil value)
Count: 1

Call Stack:
[string "=[C]"]: ?
[string "@AddOns\Blizzard_InspectUI\Localization.lua"]:8: in main chunk
[string "=[C]"]: in function `LoadAddOn'
[string "@Interface_Wrath\FrameXML\UIParent.lua"]:382: in function `UIParentLoadAddOn'
[string "@Interface_Wrath\FrameXML\UIParent.lua"]:425: in function `InspectFrame_LoadUI'
[string "@Interface_Wrath\FrameXML\UIParent.lua"]:730: in function `InspectUnit'
[string "@FrameXML\UnitPopupSharedButtonMixins.lua"]:200: in function `OnClick'
[string "@FrameXML\UnitPopupShared.lua"]:148: in function `func'
[string "@FrameXML\UIDropDownMenu.lua"]:944: in function `UIDropDownMenuButton_OnClick'
[string "*:OnClick"]:1: in function <[string "*:OnClick"]:1>
```

Temporary solution:
```
if not InspectTalentFrameSpentPoints then
	InspectTalentFrameSpentPoints = CreateFrame("Frame")
end
```

**4. Error when AchievementUI loaded.**
```
...rface\AddOns\Blizzard_AchievementUI\Localization.lua:35: attempt to index field 'tabard' (a nil value)
Count: 1

Call Stack:
[string "=[C]"]: ?
[string "@AddOns\Blizzard_AchievementUI\Localization.lua"]:35: in function `AchievementButton_Localize'
[string "@AddOns\Blizzard_AchievementUI\Localization.lua"]:56: in main chunk
[string "=[C]"]: in function `LoadAddOn'
[string "@Interface_Wrath\FrameXML\UIParent.lua"]:382: in function `UIParentLoadAddOn'
[string "@Interface_Wrath\FrameXML\UIParent.lua"]:461: in function `AchievementFrame_LoadUI'
[string "@Interface_Wrath\FrameXML\UIParent.lua"]:556: in function `ToggleAchievementFrame'
[string "TOGGLEACHIEVEMENT"]:1: in function <[string "TOGGLEACHIEVEMENT"]:1>
```

**Comment by siweia** [comment]
Issue 2 fixed in 44930.

**Comment by EKE00372** [comment]
Issue 1,2,4 were fixed.
But issue 3 still. keep from classic to cata.

**Comment by Meorawr** [comment]
Fixed in 1.15.4.56573 and 4.4.1.56574.
